src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.scss

```diff
@@ -85,6 +85,15 @@ $table-cell-height-half: $table-cell-height / 2;
     white-space: normal;
     word-break: break-all;
   }
+
+  &.kd-replicationcontrollerdetail-metric-table-cell {
+    text-align: right;
+    white-space: nowrap;
+  }
+
+  &.kd-replicationcontrollerdetail-sparkline-table-cell {
+    padding-left: 1.25 * $baseline-grid;
```

Comment:
Fixed in becde9ed735aa9274c1578b0ec70f8a2f9acc812 - however, I've added more goofy coefficients in there to size the charts (based on counting pixels in the mock) - I'm happy to relax those as well, but I don't want to do any design-on-the-fly, or step on your designer's toes.